hub/templates/digitalhub.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "databaseAccounts_digitalhub_name": {
+            "type": "SecureString"
+        },
+        "databaseAccounts_digitalhub_name_1": {
```

Comment:
Unclear how/why this differs from `_name` - is there a more descriptive name we can use for these?